src/module_wrap.cc

```diff
@@ -494,23 +493,73 @@ Maybe<uv_file> CheckFile(const URL& search,
   return Just(fd);
 }
 
+PackageConfig emptyPackage = { false, false, "" };
+std::unordered_map<std::string, PackageConfig> pjson_cache_;
```

Comment:
Are we OK with keeping this in memory for all time? (I think so, just checking, since this means more memory that's always alive in the process)